scene/gui/video_player.cpp

```diff
@@ -186,8 +186,54 @@ void VideoPlayer::_notification(int p_notification) {
 			if (texture->get_width() == 0)
 				return;
 
-			Size2 s = expand ? get_size() : texture->get_size();
```

Comment:
Hmm maybe I could re-add STRETCH_SCALE_ON_EXPAND?  Then it would be 100% consistent with TextureRect and fix the api.  Or if we're going to do a hard break and remove expand, shouldn't we remove expand from the other Controls too (TextureRect and TextureButton)?

There are already inconsistencies between the TextureButton and TextureRect implementation (texturebutton only stretches when expand is true).  I'm not sure what the desired final state is.

Reply:
If we're breaking the API, it would be good to make it consistent for all Controls yes.

If we could find a way that doesn't break the API while adding the feature and making things consistent, that would be great, but I don't really see how. I'll have a closer look at the various Controls with stretch/expand features.